test/message-generator/test/bad-pool-config-test/bad-pool-config-test.sh

```diff
@@ -0,0 +1,9 @@
+cd roles
+cargo llvm-cov --no-report -p pool_sv2
+
+cd ../utils/message-generator/
+cargo build
```

Comment:
do we really need to do `cargo build` here?
it feels redundant, since we are doing `cargo run` on the next step

Reply:
yes because we want to minimize  the risk that a test does not pass because of a timeout